src/main/java/io/aiven/klaw/service/ClusterApiService.java

```diff
@@ -91,29 +95,20 @@ public class ClusterApiService {
   @Value("${server.ssl.key-store-type:JKS}")
   private String keyStoreType;
 
-  protected static HttpComponentsClientHttpRequestFactory requestFactory;
-
-  static String URI_CREATE_TOPICS = "/topics/createTopics";
-  static String URI_UPDATE_TOPICS = "/topics/updateTopics";
-  static String URI_DELETE_TOPICS = "/topics/deleteTopics";
-
-  @Value("${klaw.jasypt.encryptor.secretkey}")
-  private String encryptorSecretKey;
-
   @Value("${klaw.clusterapi.access.username}")
   private String clusterApiUser;
 
-  @Value("${klaw.clusterapi.access.password}")
-  private String clusterApiPwd;
+  @Value("${klaw.clusterapi.access.base64.secret:null}")
```

Comment:
this is a tricky thing...
what is imply here?
value `null`
or a string containing a word like `"null"`?

Reply:
Thought setting it to empty string is better. Updated.